CONTRIBUTING.md

```diff
@@ -187,6 +187,12 @@ to this system to accept the agreement.
 
 You only need to sign the CLA once or when the CLA terms have changed.
 
+### Commit
+There is no convention for the commit message.
```

Comment:
:warning: there are conventions
we don't care of commit messages in PR but we have some when merging to master

Reply:
resolved with https://github.com/process-analytics/bpmn-visualization-js/pull/605/commits/7a84f22e488bfb434f9cf31576ba8b4e142a89db